web/src/pages/apd/proposed-budget/BudgetSummary.js

```diff
@@ -93,127 +107,146 @@ HeaderRow.propTypes = {
+      )}
 
-    <table className="budget-table" data-cy="summaryBudgetTotals">
-      <caption className="ds-h4">Activities Totals</caption>
-      <thead>
-        <tr>
-          <td className="th" id="summary-budget-null1" />
-          <th scope="col" className="ds-u-text-align--right">
-            State Total
-          </th>
-          <th scope="col" className="ds-u-text-align--right">
-            Federal Total
-          </th>
-          <th scope="col" className="ds-u-text-align--right">
-            Medicaid Total Computable
-          </th>
-        </tr>
-      </thead>
-      <tbody>
-        {Object.keys(data.combined).map(ffy => {
-          const combined = data.combined[ffy];
-          return (
-            <tr
-              key={ffy}
-              className={
-                ffy === 'total'
-                  ? 'budget-table--row__highlight budget-table--total'
-                  : ''
-              }
-            >
-              <th scope="row">{ffy === 'total' ? 'Total' : `FFY ${ffy}`}</th>
-              <td className="budget-table--number">
-                <Dollars>{combined.state}</Dollars>
-              </td>
-              <td className="budget-table--number">
-                <Dollars>{combined.federal}</Dollars>
-              </td>
-              <td className="budget-table--number">
-                <Dollars>{combined.medicaid}</Dollars>
-              </td>
-            </tr>
-          );
-        })}
-      </tbody>
-    </table>
-  </div>
-);
+      <div>
+        <h4 className="ds-h4 header-with-top-margin" aria-hidden="true">
+          MMIS Activities
+        </h4>
+        {[...years, 'total'].map(yr => (
+          <table className="budget-table" key={yr} data-cy="summaryBudgetMMIS">
+            <caption className="ds-u-visibility--screen-reader">
+              FFY {yr} MMIS Activities
+            </caption>
+            <thead>
+              <HeaderRow yr={yr} activity={'mmis'} />
+            </thead>
+            <tbody>
+              <DataRowGroup
+                data={data.mmis}
+                entries={activities.mmis}
+                year={yr}
+                apdType={apdType}
+              />
+            </tbody>
+          </table>
+        ))}
+      </div>
+
+      <table className="budget-table" data-cy="summaryBudgetTotals">
+        <caption className="ds-h4">Activities Totals</caption>
+        <thead>
+          <tr>
+            <td className="th" id="summary-budget-null1" />
+            <th scope="col" className="ds-u-text-align--right">
+              Federal Total
+            </th>
+            <th scope="col" className="ds-u-text-align--right">
+              State Total
+            </th>
+            <th scope="col" className="ds-u-text-align--right">
+              Medicaid Total Computable
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          {orderedTotals.map((rowData, index) => {
+            console.log('rowData', rowData);
```

Comment:
console log

Reply:
I thought the linter was catching these?